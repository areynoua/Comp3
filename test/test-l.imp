begin

  function @min(a, b) do
    if a <= b then
      c := a
    else
      c := b
    endif;
    return c
  end;

  function @max(a, b) do
    if a >= b then
      c := a
    else
      c := b
    endif;
    return c
  end;

  a := 5;
  print(a);
  b := @max(a, 8);
  print(b)
end
