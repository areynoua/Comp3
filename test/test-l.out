5
8
success
